lesson/day01.md

```diff
@@ -5,118 +5,105 @@ title: Day 1
+### Mac OS
+(Need process description...)
 
-In this tutorial, we will use [Brass](http://www.benryves.com/bin/brass/),
-which is a more modern assembler capable of running on all major operating
-systems. In addition, Brass can perform the linking step on its own, so we
-don't need any more programs. The next section will guide you through setting
-up Brass on your operating system- simply select from the list below.
+### Linux
+(Need process description...)
 
-ED NOTE: OS autodetection from user agent expected here, with fallback to this
-list.
+# Tools
+  ---------------------
 
-Your current OS has been automatically detected as UNKNOWN. If this is
-incorrect or you want to set things up on a different system, select the
-desired operating system from the list below.
+## Source Code Editor
 
-#### Windows
-
-#### Linux
-
-#### Mac OS X
-
-### Editor
-
-For writing source code, you will need a text editor. The source code is just
-plain text, so nearly any program will do. Some common choices are given in
-the table below, with the supported operating systems for each.
+The first tool you will need to create assembly programs is a source code editor. Some 
+popular editors are listed below along with their supported operating systems.
 
 1. Notepad
     - OS Support: Windows
-    - Included with every installation of Windows, but not many features.
+    - Included with every installation of Windows, but does not have many features.
 2. Notepad++
     - OS Support: Windows
     - Lots of features, and easy to use. 
     - Recommended for inexperienced programmers.
 3. Geany
     - OS Support: Windows, Linux
-    - Similar featureset to Notepad++ while being cross platform.
-    - only depends on GTK+ without further toolkit dependancys.
+    - Has similar features to Notepad++
+    - Only depends on GTK+ without further toolkit dependancies.
 4. Vim
     - OS Support: Windows, Mac, Linux
     - The hacker's editor. Extremely powerful, but has a very steep learning curve.
 
-ED NOTE: Editor recommendations welcome.
+**[ED NOTE]: Editor recommendations welcome.**
+
+## Assembler
 
-### Emulator
+An assembler translates source code into machine code that the calculator can execute. In
+this guide we will use an assembler called [Brass](http://www.benryves.com/bin/brass/). 
+**[CAPTAIN CALC EDIT]: Considering creating SDK for the guide that the reader can use**
 
-Once you've written a program, you need a way to run it. Simply transferring
-it to your calculator and running it there is one option (which we'll discuss
-later), but it becomes very hard to debug programs running on a calculator
-(and there will be bugs in your programs). For that reason, we'll use an
-emulator, which is a program that runs on your computer and acts like a
-calculator would.
+The following section will show how to install Brass on Windows, Mac OS, and Linux. **[CAPTAIN CALC EDIT]: The assembler should be placed in the directory for the assembler
```

Comment:
These comments should either be implemented or converted to the `[ED NOTE]` format used elsewhere.